test/e2e/framework/addon/vault/setup.go

```diff
@@ -343,6 +397,88 @@ func (v *VaultInitializer) setupRole() error {
 	return nil
 }
 
+func (v *VaultInitializer) setupKubernetes() error {
```

Comment:
nit: maybe suffix/prefix this method to make it clear you're setting up kubernetes based *auth*?